package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "novoda.github.io",
+  "version": "1.0.0",
+  "description": "Novoda Open Source Landing Page https://novoda.github.io",
+  "main": "gulpfile.js",
+  "dependencies": {},
+  "devDependencies": {
+    "gulp": "^3.9.1",
+    "gulp-coffeelint": "^0.6.0",
+    "gulp-concat": "^2.6.1",
+    "gulp-uglify": "^2.0.0",
+    "gulp-coffee": "^2.3.3",
+    "gulp-util": "^3.0.8",
+    "gulp-if": "^2.0.2",
+    "gulp-watch": "^4.3.11"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/novoda/novoda.github.io.git"
```

Comment:
I dunno :) It was auto-generated when I do `npm init` or something like that. 
I think it is not super important but I can also change to whatever you want.